packages/x-charts/src/internals/plugins/corePlugins/useChartDimensions/useChartDimensions.ts

```diff
@@ -3,13 +3,71 @@
 import * as React from 'react';
 import useEnhancedEffect from '@mui/utils/useEnhancedEffect';
 import ownerWindow from '@mui/utils/ownerWindow';
-import { DEFAULT_MARGINS } from '../../../../constants';
+import { DEFAULT_MARGINS, EMPTY_SIDES } from '../../../../constants';
 import { ChartPlugin } from '../../models';
-import { UseChartDimensionsSignature } from './useChartDimensions.types';
+import type { UseChartDimensionsSignature } from './useChartDimensions.types';
 import { selectorChartDimensionsState } from './useChartDimensions.selectors';
+import { defaultizeMargin } from '../../../defaultizeMargin';
+import { useSelector } from '../../../store/useSelector';
+// TODO: fix dep cycle, should we move selectors to their own folder/file? Eg:
+// useChartCartesianAxis/selectors/chartXAxis.ts
```

Comment:
Should we do this on the context of this PR?

Reply:
it was fixed already. I hadn't pushed 😆